dashboard/app/models/school_info.rb

```diff
@@ -133,13 +134,14 @@ def sync_from_schools
   validate :validate_with_country
   validate :validate_without_country
   validate :validate_zip
+  validate :validate_complete
 
   def usa?
     ['US', 'USA', 'United States'].include? country
   end
 
   def should_validate?
```

Comment:
Can we rename this function? `should_validate?` doesn't really explain what this function means anymore (we should be validating some things if we're doing completeness validation).

I don't have a great suggestion though. `should_do_full_validation?` maybe?